pkgs/misc/vscode-extensions/vscode-utils.nix

```diff
@@ -28,7 +28,8 @@ let
 
     installPrefix = "share/vscode/extensions/${vscodeExtUniqueId}";
 
-    buildInputs = [ unzip ] ++ buildInputs;
+    nativeBuildInputs = [ unzip ];
+    inherit buildInputs;
```

Comment:
```suggestion
```

The effect of this line was already achieved by the `remoteAttrs a [ ..... ] // {` bit above.